amida-tech/indaba-client
# Styling/inba 299 amida footer

#### What's this PR do?
Adds a footer that reads Powered by Amida to every page but the project wizard.
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-299
#### How should this be manually tested? 
deploy app
check that all pages have a footer that reads Powered by Amida with a green logo. The only place it wont show up are the pages in the wizard.
#### Any background context you want to provide?
No
#### Screenshots (if appropriate):
